devops/consuming_conan_center.rst

```diff
@@ -0,0 +1,71 @@
+.. _devops_consuming_conan_center:
+
+Using ConanCenter packages in production
+========================================
+
+ConanCenter is a fantastic resource that contains a huge knowledge base and a reference implementation of recipes contributed by the community on how to create and build Conan packages for a lot of open source third party libraries and applications supporting multiple platforms.
+
+It also provides pre-compiled binaries for a wide range of configurations: multiple OS, including Windows, Linux and OSX, a variety of compilers, compiler versions, static and shared libraries, etc. These precompiled packages can be very useful to do quick experiments, testing, onboarding, and they also serve as a proof that the recipes for those packages can work solidly for that wide range of configurations.
+
+However, in the same way modern Devops discourages to depend directly on any central repository, it is not advised to depend directly on ConanCenter for production environments, for several reasons:
+
+- ConanCenter accepts contributions that require the most recently released versions of Conan. If you are in an environment where you are unable to keep Conan up-to-date across all your systems (developers, CI, etc) - depending directly on recipes from ConanCenter carries the risk of retrieving recipes that do not work with your older version of Conan.
+- The changes in recipes might also unpexectedly cause bugs and other issues. Recipes and package updates might bump dependencies versions to use the latest, forcing a continuous update over all dependencies versions. That can cause version conflicts if your project doesn't bump dependencies versions at the same pace.
+- Reviews in ConanCenter can take a while to be reviewed and merged, as there is usually a high volume of requests. Please consider this if you may find yourself in a situation where you may need proposed changes merged with urgency. Pull Requests are considered according to priorities and resources, and will only be accepted with positive peer reviews by the Conan team.
```

Comment:
I would avoid pointing at review times. 🙊 

I would make this more generic

> If you are working with tight dead lines or on a schedule, it's far more advantageous to control when the changes you require are rolled out.

Just for the top of my head